01-intro.Rmd

```diff
@@ -184,10 +184,10 @@ Detecting existence of AWS credentials...
 Credentials found, setting appropriate configuration...
 Requesting resources from SLURM for your server...
 Submitted batch job 2733799
-Your Cromwell server is attempting to start up on **node/port gizmok30:****.  If you encounter errors, you may want to check your server logs at /home/username/cromwell-home/server-logs to see if Cromwell was unable to start up.
+Your Cromwell server is attempting to start up on node/port gizmob5:39071.  It can take up to 2 minutes prior to the port being open for use by the shiny app at https://cromwellapp.fredhutch.org or via the R package fh.wdlR. If you encounter errors, you may want to check your server logs at /fh/fast/malik_h/user/jayoung/cromwell-home/server-logs to see if Cromwell was unable to start up.
```

Comment:
We should make sure that's not a real gizmo node, nor a real path. ;)

Reply:
oops. yes.